src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Button/RadioButton_Pressed.xaml.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.Button
+{
+	[SampleControlInfo("Button", "RadioButton_Pressed", description: "RadioButtons with a lower opacity when in pressed state. Tapping on any zone should have the same result.")]
```

Comment:
I think you should indicate in comment or in description that it is currently not working on X or Y platform.